File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Commons/EvaluateCollaborationDocumentModel.ts
import BuildArchiveDocumentModel = require('App/Models/Commons/BuildArchiveDocumentModel');

interface EvaluateCollaborationDocumentModel {
    IdCollaboration: string;
    IdCollaborationVersioning: string;
    ArchiveDocument: BuildArchiveDocumentModel;
}

export = EvaluateCollaborationDocumentModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Collaborations/EvaluateCollaborationDocumentModel.cs
using System;
using VecompSoftware.DocSuiteWeb.Model.Documents;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.Collaborations
{
    public class EvaluateCollaborationDocumentModel
    {
        public Guid IdCollaboration { get; set; }
        public Guid IdCollaborationVersioning { get; set; }
        public ArchiveDocument ArchiveDocument { get; set; }
    }
}
